### DISCUSSION TITLE
Breakout room 7

### DISCUSSION THREAD

**Discussion by benmcgeachie** [IBAHCM/RPiR#263] [Archive]
Hello,

the people in Breakout room 7 are

Ben Mcgeachie - @benmcgeachie
Hana Mayall - @hanamayall
Vera Chua- @verachuachua

**Comment by benmcgeachie** [comment]
The link to my practical series 4 is below 

https://github.com/IBAHCM/benmcgeachieBCI.git

**Comment by verachuachua** [comment]
The link to my practical series 4 is:

https://github.com/IBAHCM/VeraChuaBCI.git

Thanks.

**Comment by hanamayall** [comment]
Here is the link to my practical series 4:

https://github.com/IBAHCM/hanamayallBCI
